classes/ForbiddenSeal.php

```diff
@@ -200,4 +200,18 @@ public function setBenefits() {
         $this->journal_image_display = self::$benefits[$this->level]['journal_image_display'];
         $this->avatar_size_display = self::$benefits[$this->level]['avatar_size_display'];
     }
+
+    public function calcRemainingCredit(): int {
+        $akRatio = 1;
+        switch($this->level) {
+            case 1:
+                $akRatio = 6;
```

Comment:
this isn't super intuitive how we ended up at these numbers, but I get this is `<days of seal> / <cost of seal>`. With that said maybe we rename this to `$days_per_ak` to make that more clear?

In a future PR I'd like to then make this based off whatever variable we're basing the cost off of (e.g. move forbidden seal cost definition into this class) so that if we ever change it, this stays up to date. But that's not a blocker for this one 👍